extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/util/MultiStageQueryContext.java

```diff
@@ -102,7 +102,7 @@ public class MultiStageQueryContext
 
   public static final String CTX_MAX_NUM_TASKS = "maxNumTasks";
   @VisibleForTesting
-  static final int DEFAULT_MAX_NUM_TASKS = 2;
+  public static final int DEFAULT_MAX_NUM_TASKS = 2;
```

Comment:
why public? this is still only used in `MultiStageQueryContextTest` which should be in the same package.